layouts/partials/navbar.html

```diff
@@ -3,13 +3,15 @@
     {{ $homeLink = "/" }}
 {{ end }}
 <nav class="navbar" role="navigation" aria-label="main navigation">
+    {{if not .Site.Params.dontShowLogo }}
```

Comment:
Please, add space between {{ and if

Reply:
Fixed! Should've looked closer to the code styling.